openrtb_ext/bidders.go

```diff
@@ -103,6 +103,7 @@ const (
 	BidderAJA               BidderName = "aja"
 	BidderAlgorix           BidderName = "algorix"
 	BidderAMX               BidderName = "amx"
+	BidderApacdex           BidderName = "apacdex"
```

Comment:
You need an entry for ValueImpression in this file as well for the alias to work

Reply:
I added this before
https://github.com/prebid/prebid-server/blob/01a1a4a58553e88e605052f476c190c090e5ab77/openrtb_ext/bidders.go#L177
https://github.com/prebid/prebid-server/blob/01a1a4a58553e88e605052f476c190c090e5ab77/openrtb_ext/bidders.go#L204

Reply:
Ah! You're right. My bad. Valueimpression already existed 😅